manifests/params.pp

```diff
@@ -62,24 +62,10 @@
   # for verifying the foreman https
   $foreman_proxy_foreman_ssl_ca_cert = '/etc/foreman-proxy/foreman_ssl_ca.pem'
 
-  # Puppet SSL path determination code from puppet-foreman
-  if versioncmp($::puppetversion, '4.0') < 0 {
-    $aio_package = false
-  } elsif $::rubysitedir =~ /\/opt\/puppetlabs\/puppet/ {
-    $aio_package = true
-  } else {
-    $aio_package = false
-  }
-
-  $puppet_ssldir = $aio_package ? {
-    true    => '/etc/puppetlabs/puppet/ssl',
-    default => '/etc/puppet'
-  }
-
-  $puppet_client_cert = "${puppet_ssldir}/client_cert.pem"
-  $puppet_client_key  = "${puppet_ssldir}/client_key.pem"
+  $puppet_client_cert = "${pki_dir}/puppet_client.crt"
+  $puppet_client_key  = "${pki_dir}/puppet_client.key"
```

Comment:
With our other certificates, we'd been following a convention of putting public certs into `${pki_dir}/certs` and keys into `${pki_dir}/private`.